apps/sms/services/js/conversation/conversation_service.js

```diff
@@ -0,0 +1,89 @@
+/*global ServiceMixin */
+'use strict';
+
+importScripts('../service_mixin.js');
```

Comment:
actually i'd rather use absolute if they work; I don't really know how relative work when you import a script that's not in the same directory, for example :p It's confusing. Let's use absolute !